deps.edn

```diff
@@ -494,7 +494,7 @@
     com.github.seancorfield/depstar   {:mvn/version "2.1.303"}
     expound/expound                   {:mvn/version "0.9.0"}                    ; better output of spec validation errors
     io.github.borkdude/grasp          {:mvn/version "0.1.4"}
-    io.github.clojure/tools.build     {:git/tag "v0.9.4" :git/sha "76b78fe"}
+    io.github.clojure/tools.build     {:git/tag "v0.9.5" :git/sha "24f2894"}
```

Comment:
`:mvn/version` would be easier to update. Do we prefer to load dependencies from GH?

Reply:
I think tools.build was only available via GH at first or something, there was some sort of reason why used a Git dep here. If it's available as a Maven dep now then I'm 👍 on switching to that